src/Bicep.Core.Samples/Files/InvalidParameters_LF/main.diagnostics.bicep

```diff
@@ -112,88 +111,88 @@ param objectCompletions object =
 param wrongType fluffyBunny = 'what's up doc?'
 //@[06:015) [BCP028 (Error)] Identifier "wrongType" is declared multiple times. Remove or rename the duplicates. (CodeDescription: none) |wrongType|
 //@[06:015) [no-unused-params (Warning)] Parameter "wrongType" is declared but never used. (CodeDescription: bicep core(https://aka.ms/bicep/linter/no-unused-params)) |wrongType|
-//@[16:027) [BCP031 (Error)] The parameter type is not valid. Please specify one of the following types: "array", "bool", "int", "object", "string". (CodeDescription: none) |fluffyBunny|
+//@[16:027) [BCP057 (Error)] The name "fluffyBunny" does not exist in the current context. (CodeDescription: none) |fluffyBunny|
 //@[36:037) [BCP019 (Error)] Expected a new line character at this location. (CodeDescription: none) |s|
 //@[45:046) [BCP004 (Error)] The string at this location is not terminated due to an unexpected new line character. (CodeDescription: none) |'|
 
 // invalid escape
 param wrongType fluffyBunny = 'what\s up doc?'
 //@[06:015) [BCP028 (Error)] Identifier "wrongType" is declared multiple times. Remove or rename the duplicates. (CodeDescription: none) |wrongType|
 //@[06:015) [no-unused-params (Warning)] Parameter "wrongType" is declared but never used. (CodeDescription: bicep core(https://aka.ms/bicep/linter/no-unused-params)) |wrongType|
-//@[16:027) [BCP031 (Error)] The parameter type is not valid. Please specify one of the following types: "array", "bool", "int", "object", "string". (CodeDescription: none) |fluffyBunny|
+//@[16:027) [BCP057 (Error)] The name "fluffyBunny" does not exist in the current context. (CodeDescription: none) |fluffyBunny|
```

Comment:
Worth special-casing this error? It feels like it would be easy to accidentally mix up the name & type (e.g. `param string foo`), and the error message doesn't make this obvious (compared to the previous error).